Consul.Test/CatalogTest.cs

```diff
@@ -238,5 +264,54 @@ public async Task Catalog_EnableTagOverride()
                 Assert.True(services.Response[0].ServiceEnableTagOverride);
             }
         }
+
+        [Fact]
+        public async Task Catalog_ServicesForNodes()
```

Comment:
See here (https://github.com/G-Research/consuldotnet/blob/5da7a42135bfa4e2352a57a32bbed4172324abe4/Consul.Test/AgentTest.cs#L543-L547), how to make the skip versions which do not support tested API.

Reply:
alright thanks @marcin 🚀🚀. Really learning a lot